.github/workflows/ci_servicex.yaml

```diff
@@ -51,25 +51,24 @@ jobs:
         if: ${{ matrix.app.test_required }}
         run: |
           echo "${{ matrix.app }}"
-          python -m pip install --upgrade pip setuptools wheel
-          pip install --no-cache-dir -r requirements.txt
-          pip install --no-cache-dir -r requirements_test.txt
+          python -m pip install --upgrade pip poetry
```

Comment:
We need to pin poetry itself! There is a GitHub action for installing poetry:
```
      - name: Install poetry
        uses: abatilo/actions-poetry@v2
        with:
          poetry-version: 1.2.2
```

Reply:
The CI has been updated and its working!

Reply:
<img width="1046" alt="Screenshot 2022-10-31 at 1 52 24 PM" src="https://user-images.githubusercontent.com/61829156/199087160-3a1f4a38-9dc7-4a2a-8ae4-7edf074b8c15.png">